### ISSUE TITLE
Evaluate handling of advanced CentOS/RedHat filesystem layouts

### ISSUE THREAD

**Issue by shasheene** [rescuezilla/rescuezilla#294]
CentOS8 is almost identical to RHEL8, including the installer. The only difference is around RedHat activation.

All Rescuezilla testing for advanced LVM-based systems been done using CentOS8.

This approach is completely valid, as CentOS8 and RHEL8 are identical except for RedHat's activation stuff. Rescuezilla continues to be able to backup and restore a default CentOS8/RHEL8 installation without issue.

But CentOS and RedHat have advanced partitioning modes. A user reported had an environment where the [LVM labels contained underscore characters](https://github.com/rescuezilla/rescuezilla/issues/291), breaking the scanning of Rescuezilla images. A trivial fix, but it shows there are some further edge cases beyond the basic filesystem layouts.

The task is to categorize these and if necessary, improve Rescuezilla's handling of such cases. Especially to the point of matching industry-standard Clonezilla behavior if there's any divergence. It's possible that Clonezilla itself doesn't fully handle these cases. Eg. LVMs spread over multiple disks etc.

Basically if it can be created by the CentOS/RedHat installation GUI then it should be confirmed that Rescuezilla can handle it.
